### ISSUE TITLE
Fixed Data in Stored XSS

### ISSUE THREAD

**Issue by thelicato** [hahwul/dalfox#342]
## Question
When I use the ``sxss`` mode and I set the data is there any way to set some "fixed" values? I'll provide you an example:
I've been testing **dalfox** on PortSwigger XSS Labs and there is one lab where the XSS is in the comments. The body of the POST to comment is the following:
```
csrf=<csrf_token>
name=<name>
email=<email>
comment=<comment>
website=<website>
```

Of course I want the CSRF to be fixed or the comment won't be posted. Here is the link to the Lab: https://portswigger.net/web-security/cross-site-scripting/stored/lab-html-context-nothing-encoded

Is there any way to achieve this?

## Environment
 * Dalfox Version: 2.7.0
 * Installed from: ``go install github.com/hahwul/dalfox/v2@latest``

**Comment by hahwul** [comment]
Hi @thelicato 
Thank you for submit issue and opinion 😊

Currently, there is no function to specify parameters that are not analyze and scan. 
I'll add this one(`--skip-param`)

With a little trick, you can scan only the desired parameters with the `-p` or `--param` flags. 
It's a bad case, but I think this is the best way before add the new flag.

e.g
```
$ dalfox sxss YOUR_TARGET -p name -p email -p comment -p website ~~~
```

**Comment by thelicato** [comment]
I tried the ``-p`` option and I can say that is **exactly** what I was looking for! I'll keep the issue open only because you added it to the **v2.7.1** milestone.

**Comment by hahwul** [comment]
@thelicato 
Yes, okay! Options that exclude certain parameters will definitely be useful, so I'll make sure to add them at 2.7.1! Thank you so much for the great idea :D

**Comment by hahwul** [comment]
## Flag in help
```
--ignore-param strings        Ignores this parameter when scanning.
                              * Example: --ignore-param api_token --ignore-param csrf_token
```

## Cli Test
Target,
```
$ ./dalfox url https://xss-game.appspot.com/level1/frame

[V] Triggered XSS Payload (found DOM Object): query=<sVg/onload=alert,bind()(1) class=dalfox>
    13 line:  s were found for <b><sVg/onload=alert,bind()(1) class=dalfox></b>. <a href='?'>T
[POC][V][GET][inHTML-none(1)-URL] https://xss-game.appspot.com/level1/frame?query=%3CsVg%2Fonload%3Dalert%2Cbind%28%29%281%29+class%3Ddalfox%3E
```

With `--ignore-param`
```
$ ./dalfox url https://xss-game.appspot.com/level1/frame --ignore-param query

[*] 🦊 Start scan [SID:Single] / URL: https://xss-game.appspot.com/level1/frame
[I] Found 2 testing point in DOM base parameter mining
[I] Content-Type is text/html; charset=utf-8
[*] -----------------------------------------------------------------------------------------------------------------------------
[*] [duration: 2.319873008s][issues: 0] Finish Scan!
```

## Test code
![스크린샷 2022-02-12 오후 9 37 18](https://user-images.githubusercontent.com/13212227/153711623-5b99b4e8-33f5-42c0-8e11-59cf326548c6.png)
*All pass!*

**Comment by thelicato** [comment]
That's great! Thank you
